spec/lib/trulia_spec.rb

```diff
@@ -1,60 +1,56 @@
 require 'spec_helper'
 
-RSpec.describe SherlockHomes::Trulia, sauce: true do
+RSpec.describe SherlockHomes::Trulia do
 
-  Given(:scraper) do
-    subject.base_url 'http://www.trulia.com/homes/Pennsylvania/Allentown/sold/1263178-2490-Riverbend-Rd-Allentown-PA-18103'
-    subject
+  When do
+    subject.class.set_url 'http://www.trulia.com/homes/Pennsylvania/Allentown/sold/1263178-2490-Riverbend-Rd-Allentown-PA-18103'
+    subject.load
+    subject.wait_for_estimates
```

Comment:
Should wait_for_estimates be done automatically in load, or perhaps better yet, added in dynamically when accessing estimate data?  (Trying to simplify the api for the user)